src/full/Agda/Interaction/JSONTop.hs

```diff
@@ -423,7 +423,7 @@ instance EncodeTCM Response where
     where
       encodeSolution (i, expr) = object
         [ "interactionPoint"  .= i
-        , "expression"        .= show expr
+        , "expression"        .= P.prettyShow expr
```

Comment:
Note: There might be also `prettyTCM`.

@ice1000 : Can you say something about the intended purpose of JSONification and in which syntax expressions should be placed here?

Reply:
The intended usage is to show these expressions to users, I guess. So, anything that makes the best sense to the user should be here

Reply:
The `--interaction` codepath also calls `prettyShow`, so this seems to me to be the best call for parity.